src/libPersistence/ContractStorage2.cpp

```diff
@@ -939,6 +986,7 @@ void ContractStorage2::InitTempState() {
   LOG_MARKER();
 
   t_stateDataMap.clear();
+  t_indexToBeDeleted.clear();
```

Comment:
Also need to clear in `CommitStateDB()`?

Reply:
CommitStateDB will call InitTempState()